lib/models.js

```diff
@@ -33,20 +49,15 @@ async function load(files, fn) {
 }
 
 async function applyRelations(models) {
-    if (!models || typeof models !== 'object')
-        throw new Error(`Can't apply relationships on invalid models object`);
-
+    if (typeof models !== 'object') {
```

Comment:
Why did you remove the `!models` tests? AFAICT the test is needed because if you get `models = null` this test will pass and `Object.keys()` will crash because it is given `null` as an argument. We need to test that `models` is an object but also not `null` because `typeof null` is `'object'`.

Reply:
First I removed it for best test coverage, `models` will never be `null` in this code.
As you can see [here](https://github.com/valtlfelipe/hapi-sequelizejs/pull/19/files/a67b1552c019ef7ac3af68d4d17938609b5955f9#diff-6d186b954a58d5bb740f73d84fe39073R65), the `null check` is made before the call.